### DISCUSSION TITLE
Algolia v3 upgrade

### DISCUSSION THREAD

**Discussion by okohll** [google/docsy#1317] [Q&A]
Hi, I'm trying to upgrade the search on our site to the new Algolia version as per

https://www.docsy.dev/docs/adding-content/navigation/#configure-algolia-docsearch

I've followed those instructions. The changes are here:

https://github.com/okohll/abdocs/commit/58ac229f9606c631c9b9a2bb22f06eab0fab4388

and the site itself is

https://docs.agilebase.co.uk

On loading the site now, a Javascript error occurs in the Algolia script. The error is

Uncaught TypeError: Cannot read properties of null (reading '__k')
    at Ae (js@3:8:18218)
    at js@3:8:106887
    at (index):372:32

It's possible my Algolia account code and API key are wrong. I've tried a couple of possibilities, I can't see full details of the setup in my account on algolia.com. It may be I'll have to set up a new account with them. However if that was the case I'd have hoped the error would report that.

Is there anything else obvious I'm missing?

**Comment by epi052** [comment]
from the site, it looks like this was solved. I'm having the same issue. Any advice?

**Reply by okohll** [reply]
I'm afraid I'm not sure about that specifically but I switched to Lunr, also supported by Docsy which works really well
